src/content/en/fundamentals/design-and-ui/input/touch/index.md

```diff
@@ -247,79 +138,66 @@ If, however, you expect users to interact with multiple elements at the same
 
-Touch, mouse and Pointer Events are the building blocks for adding new
-gestures into your application (see 
-[Touch, mouse and Pointer events](#touch-mouse-and-pointer-events)).
+Using these events requires calling the `addEventListener()` method on a DOM
+element, along with the name of an event, a callback function and a boolean.
+The boolean determines whether you should catch the event before or after
+other elements have had the opportunity to catch and interpret the
+events (`true` means we want the event before other elements).
 
-Include these event names in the `addEventListener()` method, along with the
-event’s callback function and a boolean. The boolean determines whether you
-should catch the event before or after other elements have had the
-opportunity to catch and interpret the events (`true` means we want the event
-before other elements).
+Here's an example of listening for the start of an interaction.
 
 <pre class="prettyprint">
 {% includecode content_path="web/fundamentals/design-and-ui/input/touch/_code/touch-demo-1.html" region_tag="addlisteners" adjust_indentation="auto" %}
 </pre>
 
-This code first checks to see if Pointer Events are supported by testing for
-`window.PointerEventsSupport`, if Pointer Events aren’t supported, we add listeners for
-touch and mouse events instead.
+Note: PointerEvents only need a single pointerdown event to handle both mouse and touch events. This is due to the design of the API.
 
-The value `window.PointerEventSupport` is determined by looking for the
-existence of `window.PointerEvent` or the now deprecated
-`window.navigator.msPointerEnabled` objects. If they are supported we use
-varibles for event names, which use the prefixed or unprefixed versions depending
-on the existence of `window.PointerEvent`.
+#### Handle Single-Element Interaction
 
-<pre class="prettyprint">
-{% includecode content_path="web/fundamentals/design-and-ui/input/touch/_code/touch-demo-1.html" region_tag="pointereventsupport" adjust_indentation="auto" %}
-</pre>
+In the short snippet of code above we only added the starting event listener,
+this wasn't just to show you how to add the event listeners. for single
+element gestures, this is the only event listener we add at first.
 
-### Handle Single-Element Interaction
-
-In the short snippet of code above you may have noticed that we only add the starting event listener, this is a conscious decision.
-
-By adding the move and end event listeners once the gesture has
-started on the element itself, the browser can check if the touch occured in a
-region with a touch event listener and if it’s not, can handle it faster
-by not having to run any additional javascript.
+By adding the "move" and "end" event listeners *after* a gesture has
+started, the browser can quickly check if the touch occurred in a
+region with an event listener and if not, it can skips calling
+any JavaScript, which can be a slow process.
```

Comment:
I'm not aware of a real performance reason to do this (it's possible that adding/removing the listeners is actually more expensive than just having them there - it's complicated).  But see below.